src/ArborX.hpp

```diff
@@ -23,6 +23,7 @@
 #include <ArborX_Exception.hpp>
 #include <ArborX_LinearBVH.hpp>
 #include <ArborX_Point.hpp>
+#include <ArborX_PredicateHelpers.hpp>
```

Comment:
Inside ArborX, we often include just `ArborX_LinearBVH.hpp` inside benchmarks and examples. Some of them use the new predicates. So, I wanted to bring them with the acceleration structures. 
On the other hand, including them just with indexes may not be sufficient if we introduce a new index and forget to include the header there. So, I wanted to make sure it's in `ArborX.hpp`.

Ideally, though, it would come with including `ArborX_Predicates.hpp`. But it would have to be included at the bottom of the file or something, as it needs to see the primitive related code.